styles/style.css

```diff
@@ -0,0 +1,65 @@
+header{
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+}
+.page-title {
```

Comment:
This can directly target the `h1` itself:

```suggestion
h1 {
```